externs/browser/html5.js

```diff
@@ -32,6 +32,11 @@
 /** @type {?HTMLSlotElement} */
 Node.prototype.assignedSlot;
 
+/**
+ * @type {string}
+ */
+Element.prototype.slot;
```

Comment:
Need a link to the documentation defining this field.

Reply:
Done